gitclient/datadog_checks/gitclient/gitclient.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+from datadog_checks.utils.subprocess_output import get_subprocess_output
+
+
+class GitclientCheck(AgentCheck):
+    def check(self, instance):
+
+        # Grab input parameters
+        git_http_repo_url = instance.get('git_http_repo_url')
+
+        # Ensure input parameters exist
+        if not git_http_repo_url:
```

Comment:
Please check for `ConfigurationError`s in `__init__` rather than the check method